### DISCUSSION TITLE
why #box(stroke:(bottom:3pt, dash: (1em, 1em))) can not compile?

### DISCUSSION THREAD

**Discussion by 1984204066** [typst/typst#2976] [Q&A] [answered]
#box(stroke:(bottom:3pt)) 
and 
#box(stroke:(dash:(1pt,1pt)))
can work, but 
#box(stroke:(bottom:3pt, dash: (1em, 1em))) does not!!!

why?

**Comment by Enter-tainer** [comment, marked as answer]
is this what you need?

```
#box(stroke:(bottom: (thickness: 1pt, dash: (1em, 1em))))
```

**Reply by 1984204066** [reply]
great!! it works now!! thank you!!
